src/template.yml

```
@@ -17,12 +17,12 @@ Metadata:
     Labels:
       ["adf", "aws-deployment-framework", "multi-account", "cicd", "devops"]
     HomePageUrl: https://github.com/awslabs/aws-deployment-framework
-    SemanticVersion: 3.1.2
+    SemanticVersion: 3.4.1
```

This should not be changed in this PR:
```suggestion
    SemanticVersion: 3.1.2
```

---

Woops, changed this for deploying locally